tck/features/clauses/call/Call5.feature

```diff
@@ -29,3 +29,153 @@
+    When executing query:
+      """
+      CALL test.my.proc(null) YIELD out AS a
+      RETURN a
+      """
+    Then the result should be, in order:
+      | a     |
+      | 'nix' |
+    And no side effects
+
+  Scenario: [3] Explicit procedure result projection with RETURN *
+    Given an empty graph
+    And there exists a procedure test.my.proc(in :: INTEGER?) :: (out :: STRING?):
+      | in   | out   |
+      | null | 'nix' |
+    When executing query:
+      """
+      CALL test.my.proc(null) YIELD out
+      RETURN *
+      """
+    Then the result should be, in order:
+      | out   |
+      | 'nix' |
+    And no side effects
+
+  Scenario Outline: [4] The order of yield items is irrelevant
+    Given an empty graph
+    And there exists a procedure test.my.proc(in :: INTEGER?) :: (A :: INTEGER?, B :: INTEGER?) :
+      | in   | a | b |
+      | null | 1 | 2 |
+    When executing query:
+      """
+      CALL test.my.proc(null) YIELD <yield>
+      RETURN a, b
+      """
+    Then the result should be, in order:
+      | a | b |
+      | 1 | 2 |
+    And no side effects
+
+    Examples:
+      | yield |
+      | a, b  |
+      | b, a  |
+
+  Scenario Outline: [5] Rename Single output
+    Given an empty graph
+    And there exists a procedure test.my.proc(in :: INTEGER?) :: (A :: INTEGER?, B :: INTEGER?) :
+      | in   | a | b |
+      | null | 1 | 2 |
+    When executing query:
+      """
+      CALL test.my.proc(null) YIELD a, b as c
+      RETURN a, c
+      """
+    Then the result should be, in order:
+      | a | c |
+      | 1 | 2 |
+    And no side effects
+
+  Scenario Outline: [6] Rename all outputs
+    Given an empty graph
+    And there exists a procedure test.my.proc(in :: INTEGER?) :: (A :: INTEGER?, B :: INTEGER?) :
+      | in   | a | b |
+      | null | 1 | 2 |
+    When executing query:
+      """
+      CALL test.my.proc(null) YIELD a as b, b as d
+      RETURN b, d
+      """
+    Then the result should be, in order:
+      | b | d |
+      | 1 | 2 |
+    And no side effects
+
+  Scenario Outline: [7] Rename single output to an unbound variable name
+    Given an empty graph
+    And there exists a procedure test.my.proc(in :: INTEGER?) :: (A :: INTEGER?, B :: INTEGER?) :
+      | in   | a | b |
+      | null | 1 | 2 |
+    When executing query:
+      """
+      CALL test.my.proc(null) YIELD a as b, b as c
+      RETURN b, c
+      """
+    Then the result should be, in order:
+      | b | c |
+      | 1 | 2 |
+    And no side effects
+
+  Scenario Outline: [8] Rename all outputs to unbound variable names
+    Given an empty graph
+    And there exists a procedure test.my.proc(in :: INTEGER?) :: (A :: INTEGER?, B :: INTEGER?) :
+      | in   | a | b |
+      | null | 1 | 2 |
+    When executing query:
+      """
+      CALL test.my.proc(null) YIELD a as b, b as a
+      RETURN b, a
+      """
+    Then the result should be, in order:
+      | b | a |
+      | 1 | 2 |
+    And no side effects
```

Comment:
The renaming scenario could be integrated into a scenario outline, I think:

```suggestion
  Scenario Outline: [6] Rename outputs to unbound variable names
    Given an empty graph
    And there exists a procedure test.my.proc(in :: INTEGER?) :: (A :: INTEGER?, B :: INTEGER?) :
      | in   | a | b |
      | null | 1 | 2 |
    When executing query:
      """
      CALL test.my.proc(null) YIELD <rename>
      RETURN <a>, <b>
      """
    Then the result should be, in order:
      | <a> | <b> |
      | 1   | 2   |
    And no side effects
    
    Examples:
      | <rename> | <a> | <b> |
      | a AS c, b AS d | c   | d   |
      | a AS b, b AS d | b   | d   |
      | a AS c, b AS a | c   | a   |
      | a AS b, b AS a | b   | a   |
      | a AS c, b AS b | c   | b   |
      | a AS c, b      | c   | b   |
      | a AS a, b AS d | a   | d   |
      | a, b AS d      | a   | d   |
      | a AS a, b AS b | a   | b   |
      | a AS a, b      | a   | b   |
      | a, b AS b      | a   | b   |
    
```

As illustrated, this makes iterating through a larger number of case very simple.

Similarly a the corresponding failing scenarios can be arrange in a single scenario outline.